SC_redesign/Crop_and_Soil/tests_SC/soil_tests/test_residue_partition.py

```diff
@@ -93,3 +94,94 @@ def test_determine_plant_metabolic_active_carbon_usage(decomposition_moisture_ef
     assert expected == ResiduePartition._determine_plant_metabolic_active_carbon_usage(decomposition_moisture_effect,
                                                                                        decomposition_temperature_effect,
                                                                                        plant_metabolic_carbon_amount)
+
+
+@pytest.mark.parametrize("plant_metabolic_carbon_amount, tillage_fraction", [
+    (3, 0.4),  # default value
+    (50, 0.5),  # higher value
+    (1.8, 0.33),  # arbitrary values
+    (2.2, 0)  # zeros
```

Comment:
These comments are a bit confusing. If you're going to have comments on cases, make sure useful to the reader. It is also helpful to include more cases that change things one-at-a-time:

```suggestion
    (3, 0.4),  # default
    (50, 0.4),  # increased carbon
    (3, 1.0),  # increased tillage
    (1.8, 0.33),  # decreased carbon & tillage
    (0, 0.4),  # no carbon
    (3, 0),  # no tillage
    (0, 0),  # neither
```

Consider this in all your tests.